Frends.Community.SQL.Tests/SQLTaskTest.cs

```diff
@@ -1,8 +1,17 @@
 using System;
+using System.Collections.Generic;
 using System.Data;
 using System.Diagnostics;
 using System.IO;
+using System.Threading.Tasks;
+using System.Threading;
 using NUnit.Framework;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Newtonsoft.Json.Linq;
```

Comment:
You are not using this namespace, so it can be removed.

Reply:
Removed namespace while separating test class to different file.